k8s/namespaces/ccd/message-publisher/demo.yaml

```diff
@@ -2,25 +2,9 @@ apiVersion: helm.fluxcd.io/v1
 kind: HelmRelease
 metadata:
   name: message-publisher
-  annotations:
-    fluxcd.io/automated: "true"
-    fluxcd.io/tag.java: glob:pr-74-*
-    hmcts.github.com/prod-automated: disabled
```

Comment:
Could we possibly keep these but change to prod please, just so it's easier to edit next time we want to deploy a different image too. We should then keep lines 10 and 26 (with prod image)